src/module.c

```diff
@@ -5358,7 +5358,12 @@ int RM_HashSet(RedisModuleKey *key, int flags, ...) {
  * expecting a RedisModuleString pointer to pointer, the function just
  * reports if the field exists or not and expects an integer pointer
  * as the second element of each pair.
- *
+ * 
+ * REDISMODULE_HASH_EXPIRE_TIME: retrieves the expiration time of a field in the hash.
```

Comment:
let's document which is the first version that supports this flag.

maybe we even want to add an API like RedisModule_GetOpenKeyModesAll so the user can dynamically detect which flags are supported?
i suppose we can live without it.

Reply:
Let's live without it. for now.